### ISSUE TITLE
[Feature request] Allow blocking trades of certain items

### ISSUE THREAD

**Issue by darorl89** [lemasato/POE-Trades-Companion#108]
Hey, poe.trade is lagging really hard. When it does not update on time and you post something underpriced, there will be constant PMs for the few hours that you can only get rid of with /dnd. I guess it's not possible to block PMs, but we could at least block trade companion popup for this item, thus allowing to trade normally without being distracted all the time.

**Comment by lemasato** [comment]
Yup, being able to ignore whispers for a specific item will be part of 1.13.

**Comment by deduijk** [comment]
Maybe not just an ignore, but maybe also the option to send an automatic response (item already sold / no longer listed or something along those lines)?

**Comment by lemasato** [comment]
Automatic responses would be out of GGG terms.
Anyways, if you were mapping, and someone send a whisper for an ignored item, it could be dangerous/deadly if the tool suddenly opens the chat to write a message.

**Comment by deduijk** [comment]
True, hadn't considered that.
